### PR TITLE
Resources: Fix font-size for demos

### LINKED ISSUE #110: Remove font-size: 62.5% in demos
http://bugs.jqueryui.com/ticket/10131

### PR DESCRIPTION
**Pull request by jzaefferer**
Fixes #110
Ref jquery/jquery-wp-content#380

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -162,31 +166,45 @@ grunt.registerTask( "build-demos", function() {
...
-			/<script src="\.\.\/\.\.\/external\/jquery\/jquery\.js">/,
-			"<script src=\"//code.jquery.com/jquery-" + jqueryCore + ".js\">" );
+	function deAmd( $, destDir ) {
+		var i18n, globalize,
+			bootstrap = $( "script[src='../bootstrap.js']" ),
+			require = $( "script[src='../../external/requirejs/require.js']" ),
+			extra = bootstrap.attr( "data-modules" );
+
+		// Replace the src on the first tag with core
+		require
```
Comment (scottgonzalez) [inline Gruntfile.js]: Seems like you don't need a line break here.

Comment (jzaefferer) [inline Gruntfile.js]: That isn't supposed to be a part of this PR.

**Comment by jzaefferer** [comment]
Updated, now only contains the CSS changes.

**Comment by scottgonzalez** [comment]
Looks good.

**Pull request merged**